Peer.Domain/Models/PeerPullRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Peer.Domain.Models
+{
+    public class PeerPullRequest
```

Comment:
nit: Unless there's another `*PullRequest` type I'd probably drop the `Peer` prefix; it reads like a distinction (to me).